### DISCUSSION TITLE
graphiql: Where did the `?query=` string in the URL go? 🤔

### DISCUSSION THREAD

**Discussion by mfn** [graphql/graphiql#3094] [Q&A / Support]
Sorry for this stupid question but I'm kind of lost.

So far I've been using a quite old version, seems from reading the source of the HTML it references `https://cdnjs.cloudflare.com/ajax/libs/graphiql/0.13.0/graphiql.min.js` so there's that.

When I use this version, there is the `?query=` automatically updated when I write a query:
![image](https://user-images.githubusercontent.com/87493/223700952-731cd89a-4d8e-4c80-8300-0a4476217e2e.png)

This is extremely useful and often-used within our teams because, well, easy 😅

I'm now finally upgrading to what I think is a more / the most recent version which references e.g. `//unpkg.com/graphiql/graphiql.min.js` which expands to `https://unpkg.com/graphiql@2.4.0/graphiql.min.js`
![image](https://user-images.githubusercontent.com/87493/223701887-9813deb7-44b7-4de8-9028-a2a2481daa98.png)

But I can't find this feature anymore. Is it gone or did I miss it somewhere?

During writing this question I found https://github.com/graphql/graphiql/pull/1579 so I'm led to believe after some major version rewrite it hasn't made it in yet? (also found https://github.com/graphql/graphiql/discussions/978 which seems to talk about this)

Do we know if there's a workaround which ideally can be applied at the HTML level locally?

thank you and sorry to bother you 🙏🏼

**Comment by LekoArts** [comment]
afaik you have to implement it on your own for now. We did it like this: https://github.com/gatsbyjs/gatsby/blob/7fd887e23f5f1a96489d19cb3e71b045f8ad1512/packages/gatsby-graphiql-explorer/src/app.jsx
